apps/site/pages/docs/customizing/components.mdx

```diff
@@ -163,6 +163,97 @@ To ensure visual consistency with your store's theme, we highly recommend utiliz
 
 ---
 
+## Importing FastStore UI component styles
+
+If you are using a [FastStore UI component](/components) and want to apply its styles to your 
+FastStore project, follow the steps below. In this example, we will override the 
+[`ProductTitle`]() component within the [`ProductDetails`]() section. 
+
+### Before you begin
+As the [Customizing a new component when using override](/docs/customizing/components#customizing-a-new-component-when-using-override)
+you should also have a basic understanding of [overriding native components and props](/docs/building-sections/overriding-components-and-props).
+Additionally, we recommend to use [CSS Module](https://nextjs.org/docs/app/building-your-application/styling/css-modules) for styling your new component.
+
+### Step-by-step
+
+#### Step 1 - Creating the stylesheet
+Create the stylesheet file and export the component 
+
+1. Open your FastStore project and navigate to the `components`, 
```

Comment:
I think it makes more sense `Creating the custom component` to be the step 1, and then create the stylesheet file.